Gemfile

```diff
@@ -1,3 +1,9 @@
-source 'http://rubygems.org'
+source 'https://rubygems.org'
 
-gemspec
\ No newline at end of file
+gemspec
+gem 'rspec', '~> 3.5'
+gem 'rubocop', group: 'development'
+
+group :test do
+  gem 'simplecov', '~> 0.21.2'
```

Comment:
Any reason this needs to be version restricted? I know it can be scary to let it float, but I'd rather have version restrictions only when we _know_ that it can't be upgraded.